.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Use this template for reporting bugs. Please search existing issues first.
+labels: bug
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Environment
+<!--- Output from `k6 version`, e.g. `k6 v0.26.0-dev (dev build, go1.13.4, linux/amd64)` -->
```

Comment:
maybe we should request some more details here? for example, their OS (in case they're using older versions of k6 that don't emit that in `k6 version`) and whether they're using docker containers, since that also won't be shown in `k6 version`?

Reply:
Makes sense. See d31d67cd.